cristin-import/src/test/resources/features/ChapterArticleRules.feature

```diff
@@ -1,19 +1,19 @@
 Feature:
 
-  Scenario: Map returns NVA Resource with Reference having a PublicationInstance of type
-  ChapterArticle when the Cristin Result's secondary category is "Kapittel"
+  Scenario: Cristin Result of type "Academic chapter/article/Conference paper" maps to NVA entry of type
+  "ChapterArticle"
```

Comment:
I believe this scenario description gives a false impression of what the test actually proves. It claims that "Academic chapter/article/Conference paper" gets mapped, but in reality it only proves this for chapter

Reply:
Academic chapter/article/Conference is the English description of the type Kapittel accroding to the excel file in Confluence

Reply:
Aaaah, my bad